netbox/dcim/api/views.py

```diff
@@ -402,7 +401,18 @@ def napalm(self, request, pk):
         # Connect to the device
```

Comment:
Minor nit, but this comment should be moved down to the driver initialization since we've inserted a new comment below it.

Reply:
Updated the comment location. I then also moved two other variable assignments to below the logic to assist in the aesthetics of the code and where the work is getting completed.